app/models/menu.rb

```diff
@@ -1,5 +1,5 @@
 class Menu < ApplicationRecord
   validates_presence_of :name
-
+  belongs_to :restaurant
```

Comment:
Yes, it does. It was not practical to put this off. This story could not be delivered without this association in place.